### ISSUE TITLE
Nothing is visible

### ISSUE THREAD

**Issue by robbi-blechdose** [tommy1019/Totality#2]
I'm using this code here:
`GameController gameController = new GameController();
        
        gameController.addControllerElement(new Button("fire", 0, 0, 0.2f, 0.1f));
        gameController.addControllerElement(new Button("cycleWeaponGroup", 0.1f, 0, 0.2f, 0.1f));
        
        gameController.addControllerElement(new Joystick("movement", 0, 0.5f, 0.1f, 0.1f));
        
        Totality.instance.setDefaultController(gameController);
        
        Totality.instance.start();`

The webserver starts and I can connect, but the screen is blank. Upon checking with the website inspector, it seems that stuff like opacity, height, and width are zero.

**Comment by tommy1019** [comment]
Best I can tell, your controller elements are outside of the window.

Keep in mind, the position given is the center of the element. Try this code for configuring your default controller.

```
gameController.addControllerElement(new Button("fire", 0.2f, 0.2f, 0.2f, 0.1f));
gameController.addControllerElement(new Button("cycleWeaponGroup", 0.1f, 0.1f, 0.2f, 0.1f));
	    
gameController.addControllerElement(new Joystick("movement", 0.5f, 0.5f, 0.1f, 0.1f));
```

**Comment by robbi-blechdose** [comment]
Thanks for the answer, but it still doesn't work. This is what the inspector is giving me:
`<body style="-moz-user-select: none; overflow: hidden; height: 100%; width: 100%; position: fixed;"><div id="fire" style="background: rgba(0, 0, 0, 0) none repeat scroll 0% 0%; border: medium none; outline: medium none currentcolor; position: absolute; left: 256px; top: 0px; width: 0px; height: 0px; opacity: 0;"></div><canvas id="firecanvas" class=" canvasClass" width="0" height="0" style="width: 0px; height: 0px; top: 0px; left: 256px; position: absolute; z-index: -10;"></canvas><div id="cycleWeaponGroup" style="background: rgba(0, 0, 0, 0) none repeat scroll 0% 0%; border: medium none; outline: medium none currentcolor; position: absolute; left: 128px; top: 0px; width: 0px; height: 0px; opacity: 0;"></div><canvas id="cycleWeaponGroupcanvas" class=" canvasClass" width="0" height="0" style="width: 0px; height: 0px; top: 0px; left: 128px; position: absolute; z-index: -10;"></canvas><div id="movementdiv" style="background: rgba(0, 0, 0, 0) none repeat scroll 0% 0%; border: medium none; outline: medium none currentcolor; position: absolute; left: 128px; top: 0px; touch-action: none;"><div class="nipple collection_0" id="nipple_0_0" style="position: absolute; opacity: 0.5; display: block; z-index: 999; transition: opacity 250ms ease 0s;"><div class="back" style="position: absolute; display: block; width: 0px; height: 0px; margin-left: 0px; margin-top: 0px; background: blue none repeat scroll 0% 0%; opacity: 0.5; border-radius: 50%;"></div><div class="front" style="width: 0px; height: 0px; position: absolute; display: block; margin-left: 0px; margin-top: 0px; background: blue none repeat scroll 0% 0%; opacity: 0.5; border-radius: 50%;"></div></div></div></body>
`
As you can see, the color is white, and width, height, opacity are zero.

**Comment by tommy1019** [comment]
Have you included gson in your project?

If you have and it is still not working, can you post?
 - The Totality log from console
 -What browser you are using.

**Comment by robbi-blechdose** [comment]
I didn't have it, added it, problem persists.
I'm using firefox (58.0.2), and here's the log:
`[Totality server] Starting
[Totality server] Local ip: 192.168.2.115
[Totality server] Started web socket server.
[Totality server] Started web server.`

**Comment by gcmvy3** [comment]
Hey - I managed to recreate the error. It seems to be caused by the Windows version of Firefox. I'm working on a fix right now, but in the meantime you could try it with another browser.

**Comment by tommy1019** [comment]
Fixed in 1.0.1
